Cargo.toml

```diff
@@ -1,12 +1,14 @@
 [package]
 name = "pulldown-cmark"
-version = "0.2.0"
+# Note: This not a valid release, intended to be filled in when it's ready.
+version = "0.0.0"
```

Comment:
I'm wondering whether this should be changed to "0.3.0" in this PR.

Reply:
Let's do it in a separate PR in a day or so?